lazy_tensor_core/lazy_tensor_core/csrc/ops/scalar.h

```diff
@@ -31,6 +31,16 @@ torch::lazy::hash_t ScalarHash(const at::Scalar& s);
 
 using at::operator<<;
 
+inline torch::lazy::NodePtr ScalarOp(const at::Scalar& value,
```

Comment:
Do we need all of the helpers? And you probably don't need inline as the function is defined in the header.